### ISSUE TITLE
Another rate limit encountered

### ISSUE THREAD

**Issue by JonSchram** [crunchy-labs/crunchy-cli#368]
**Describe the bug**
Cannot download any episodes from The Daily Life of the Immortal King. I've tried using the archive command and the download command, and the result is the same.

**To Reproduce**
```
$ crunchy archive --audio zh-CN "https://www.crunchyroll.com/series/GZJH3DJ8E/the-daily-life-of-the-immortal-king[S4]"
```
Output:
```
:: ✔ Logging in
:: ✔ Parsed url 1
:: ✔ Loaded series information for url 1
:: The Daily Life of the Immortal King Season 4 (The Daily Life of the Immortal King (Season 4))
::      1. Way-of-Heaven-level Swordsmanship » S04E01
::      2. Snack Noodle in Collector's Edition » S04E02
::      3. Resonance Between Masters » S04E03
::      4. Underground Spiritual Vein » S04E04
::      5. Wang Ling's Art of Great Luck » S04E05
::      6. It's All Your Fault » S04E06
::      7. Art of Explosion? » S04E07
::      8. Defending the Faction 60 » S04E08
::      9. Rong Is in Dander » S04E09
::      10. Start Immortal King Mode » S04E10
::      11. Birth of Evil Sword Lord » S04E11
::      12. War between True Immortals » S04E12
:: An error occurred: You've probably hit a rate limit. Try again later, generally after 10-20 minutes the rate limit is over and you can continue to use the cli (https://cr-play-service.prd.crunchyrollsvc.com/v1/manifest/GWDU8J388/...)
```

**Expected behavior**
I expected this to download the episodes.

**Screenshots**
n/a

**Client (please complete the following information):**
 - OS: Windows
 - Version: v3.3.5

**Additional context**
I know that #365 was fixed in v3.3.5, and this bug seems very similar, yet I am still encountering this issue.
This series only has English, Hinidi, and Chinese audio, so maybe something assumes there will be Japanese audio?

**Comment by DanielPfe** [comment]
I also noticed this issue with some episodes. Ancient magus‘ bride season 2 episode 20 & 21 in my case. Others had no issue.

**Comment by bytedream** [comment]
Fixed in [v3.4.0](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.4.0). Crunchyroll has recently added a new format how fps are represented, this caused the false-positive error
